### PR TITLE
Fix vite

### PR DESCRIPTION
**Pull request by furqanrydhan**
Trying to fix vite builds, moving multihash to non lazy seemed to help + cbor-x instead of cbor-web fixed the main set of issues. 

Have no idea on regression or other frameworks

### CODE REVIEW

File: packages/sdk/package.json [hunk 1]
```diff
@@ -113,6 +113,7 @@
     "bn.js": "^5.2.1",
     "bs58": "^5.0.0",
     "cbor-web": "^8.1.0",
+    "cbor-x": "^1.4.1",
```
Comment (joaquim-verges) [inline packages/sdk/package.json:116]: cbor-x looks pretty good, we can remove cbor-web then right?

File: packages/sdk/src/evm/common/feature-detection.ts [hunk 2]
```diff
@@ -294,14 +295,15 @@ async function extractIPFSHashFromBytecode(
   );
 
   // load these lazily to avoid loading them when they are not needed
-  const [cbor, multiHashes] = await Promise.all([
-    import("cbor-web"),
-    import("multihashes"),
-  ]);
+  const [cbor] = await Promise.all([import("cbor-x")]);
```
Comment (joaquim-verges) [inline packages/sdk/src/evm/common/feature-detection.ts:298]: Prob don't need a promise all anymore here. cbor-x allows to import just the decoding too:

`import { decode } from 'cbor-x/decode'`

File: packages/sdk/src/evm/common/feature-detection.ts [hunk 3]
```diff
@@ -23,6 +23,7 @@ import {
 import { ExtensionNotImplementedError } from "./error";
 import { ThirdwebStorage } from "@thirdweb-dev/storage";
 import { BaseContract, BigNumber, ethers } from "ethers";
+import multihashes from "multihashes";
```
Comment (joaquim-verges) [inline packages/sdk/src/evm/common/feature-detection.ts:26]: weird that this changed anything

File: packages/sdk/package.json (hunk 1 above)
Comment (furqanrydhan) [reply packages/sdk/package.json:115]: done in latest

File: packages/sdk/src/evm/common/feature-detection.ts (hunk 2 above)
Comment (furqanrydhan) [reply packages/sdk/src/evm/common/feature-detection.ts]: removed it and moved cbor to top

File: packages/sdk/src/evm/common/feature-detection.ts (hunk 3 above)
Comment (furqanrydhan) [reply packages/sdk/src/evm/common/feature-detection.ts]: agree, i'd love someone else to verify this was needed.

**Comment by jnsdls** [comment]
going to take this on this afternoon

**Pull request merged**